File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthRequestParams.java
package com.braintreepayments.api.venmo;

import androidx.activity.ComponentActivity;

import com.braintreepayments.api.BrowserSwitchOptions;

/**
 * Used to request Venmo authentication via {@link VenmoLauncher#launch(ComponentActivity, VenmoPaymentAuthRequest.ReadyToLaunch)} )}
 */
public class VenmoPaymentAuthRequestParams {

    private BrowserSwitchOptions browserSwitchOptions;

    VenmoPaymentAuthRequestParams(BrowserSwitchOptions browserSwitchOptions) {
        this.browserSwitchOptions = browserSwitchOptions;
    }

    BrowserSwitchOptions getBrowserSwitchOptions() {
        return browserSwitchOptions;
    }
}

File: Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthRequestParams.kt
package com.braintreepayments.api.venmo

import androidx.annotation.RestrictTo
import com.braintreepayments.api.BrowserSwitchOptions

/**
 * Used to request Venmo authentication via [VenmoLauncher.launch] )}
 */
@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
data class VenmoPaymentAuthRequestParams internal constructor(
    val browserSwitchOptions: BrowserSwitchOptions
)
